src/backend/oracle-data-api/src/main/resources/data.sql

```diff
@@ -70,26 +84,60 @@ INSERT INTO JJDISPUTED_COUNT (ID, CREATED_BY, CREATED_TS, MODIFIED_BY, MODIFIED_
+(26, 'System', LOCALTIMESTAMP, 'System', LOCALTIMESTAMP, 'EA01288745', 'EA01288745', 1, 'N', '', '146(3) Speed against highway sign', 0, 'Y', 196.00, 196.00, 'Y', 'N', DATEADD('DAY', 30, CURRENT_DATE), null),
+(27, 'System', LOCALTIMESTAMP, 'System', LOCALTIMESTAMP, 'EA03665215', 'EA03665215', 2, 'N', '', '146(3) Speed against highway sign', 0, 'Y', 368.00, 368.00, 'Y', 'Y', DATEADD('DAY', 30, CURRENT_DATE), null),
 (28, 'System', LOCALTIMESTAMP, 'System', LOCALTIMESTAMP, 'EA00241528', 'EA00241528', 1, 'Y', '', '92.1(1) MVA - Fail to stop resulting in pursuit', 1, 'N', 380.00, 380.00, 'Y', 'N', DATEADD('DAY', 30, CURRENT_DATE), null),
 (29, 'System', LOCALTIMESTAMP, 'System', LOCALTIMESTAMP, 'EA00241528', 'EA00241528', 2, 'Y', '', '146(5) MVA 0 Speed against area sign', 1, 'Y', 196.00, 196.00, 'Y', 'N', DATEADD('DAY', 30, CURRENT_DATE), null),
-(30, 'System', LOCALTIMESTAMP, 'System', LOCALTIMESTAMP, 'EA00241528', 'EA00241528', 3, 'Y', '', '92.1(1) MVA - Fail to stop resulting in pursuit', 1, 'N', 380.00, 380.00, 'Y', 'N', DATEADD('DAY', 30, CURRENT_DATE), null);
+(30, 'System', LOCALTIMESTAMP, 'System', LOCALTIMESTAMP, 'EA00241528', 'EA00241528', 3, 'Y', '', '92.1(1) MVA - Fail to stop resulting in pursuit', 1, 'N', 380.00, 380.00, 'Y', 'N', DATEADD('DAY', 30, CURRENT_DATE), null),
+(31, 'System', LOCALTIMESTAMP, 'System', LOCALTIMESTAMP, 'EA07789632', 'EA07789632', 1, 'Y', '', '146(1) Speed in/outside Municipality', 0, 'N', 196.00, 196.00, 'Y', 'N', DATEADD('DAY', 30, CURRENT_DATE), null),
+(32, 'System', LOCALTIMESTAMP, 'System', LOCALTIMESTAMP, 'EA01115462', 'EA01115462', 1, 'Y', '', '146(3) Speed against highway sign', 0, 'N', 196.00, 196.00, 'Y', 'N', DATEADD('DAY', 30, CURRENT_DATE), null),
+(33, 'System', LOCALTIMESTAMP, 'System', LOCALTIMESTAMP, 'EA04788953', 'EA04788953', 1, 'Y', '', '146(3) Speed against highway sign', 0, 'N', 196.00, 196.00, 'Y', 'N', DATEADD('DAY', 30, CURRENT_DATE), null),
+(34, 'System', LOCALTIMESTAMP, 'System', LOCALTIMESTAMP, 'EA04788953', 'EA04788953', 2, 'Y', '', '214.2(1) Using Electronic device while driving', 0, 'N', 368.00, 368.00, 'Y', 'N', DATEADD('DAY', 30, CURRENT_DATE), null),
+(35, 'System', LOCALTIMESTAMP, 'System', LOCALTIMESTAMP, 'EA04788953', 'EA04788953', 3, 'Y', '', '214.2(2) Emailing or texting while driving', 1, 'N', 368.00, 368.00, 'Y', 'N', DATEADD('DAY', 30, CURRENT_DATE), null),
+(36, 'System', LOCALTIMESTAMP, 'System', LOCALTIMESTAMP, 'EA05888444', 'EA05888444', 1, 'Y', '', '146(3) Speed against highway sign', 0, 'N', 138.00, 138.00, 'Y', 'N', DATEADD('DAY', 30, CURRENT_DATE), null),
+(37, 'System', LOCALTIMESTAMP, 'System', LOCALTIMESTAMP, 'AO38375806', 'AO38375806', 1, 'Y', '', '194(3) Ride motorcycle without required helmet', 1, 'N', 138.00, 138.00, 'Y', 'N', DATEADD('DAY', 30, CURRENT_DATE), null),
```

Comment:
It looks like the statutes are different in this data than what's in JUSTIN. This id "194(3)" refers to "Permit unlawfully seated passenger", not "Ride motorcycle without required helmet"